app/assets/themes/fearlesscities/_processes.scss

```diff
@@ -0,0 +1,31 @@
+@import "variables";
+
+.process-nav {
+  .process-nav__content ul {
+    padding: 1em 0 1em 0;
+
+    li {
+      box-shadow: none;
+
+      &.is-active {
+        box-shadow: none;
+        background-color: $fearless-red;
+        border: 1px solid $fearless-black;
```

Comment:
I would apply this thin black border to other sections as well (you can use [this file](https://github.com/Platoniq/decidim-pemb/blob/main/app/assets/stylesheets/pemb/partials/_borders.scss) as reference for what classes have these borders applied)